resources/views/admin/ilog_add.blade.php

```diff
@@ -1,5 +1,9 @@
 @extends('layouts.app')
 
+@section('page_title')
+    {{ config('app.name')}} - {{ "Log Interaction" }}
```

Comment:
Would it not be better to move `{{ config('app.name')}} -` into `global/head.blade.php`? Looks to be the same on every page.
e.g.
```
<title>{{ config('app.name')}} - @yield('page_title')</title>
```

Reply:
Changed in all affected files and commited.